packages/desktop-electron/index.ts

```diff
@@ -113,7 +111,6 @@ async function createWindow() {
       nodeIntegrationInWorker: false,
       nodeIntegrationInSubFrames: false,
       contextIsolation: true,
-      enableRemoteModule: false,
```

Comment:
Removed in since electron v14. Typescript picked it up:

https://www.electronjs.org/docs/latest/breaking-changes#removed-remote-module